pkgs/tools/text/ispell/default.nix

```diff
@@ -37,11 +41,52 @@ stdenv.mkDerivation rec {
     #define MINIMENU
     #define HAS_RENAME
     EOF
-
   '';
+
   preBuild = ''
     for dir in $out/share/emacs/site-lisp $out/share/info $out/share/man/man1 $out/share/man/man4 $out/bin $out/lib; do
     mkdir -p $dir
     done
   '';
+
+  meta = with stdenv.lib; {
+    description = "An interactive spell-checking program for Unix";
+    homepage = "https://www.cs.hmc.edu/~geoff/ispell.html";
+    license = ''
```

Comment:
Unfortunately the type of `meta` attributes is not formally specified, but `license` is generally expected to be an attrset of the form used in `lib/licenses.nix`. We have the catchall `free` license with the name "Unspecified free software license". I think it would be fine to apply it here (since the license is clearly free but custom) and provide a link to the full license in a comment. Alternatively you could come up with your own attrset and set the "url" appropriately (but you'd need to come up with a name for the license).